templates/base.html

```diff
@@ -34,7 +34,11 @@ <h2>Make Connections</h2>
                 <p>Click on any edge to explore connections.</p>
                 <form class="upload-form" method="post" id="upload-form" encType="multipart/form-data">
                     {% csrf_token %}
-                    <input class="file-input-box" type="file" name="{{ upload_form.file_field.html_name }}" id="upload-box" multiple/>
+                    <input type="file"
+                           class="file-input-box"
+                           name="{{ upload_form.file_field.html_name }}"
+                           id="upload-box"
+                           webkitdirectory mozdirectory msdirectory odirectory directory multiple />
```

Comment:
wow good to know there are so many folder types.. I can select folders with no issues in my local. good job! but not multiple files. maybe we dont need it anymore?

Reply:
I don't think so. They get one or the other; file selection or folder selection. Jane asked for folder selection.